x-pack/plugins/ml/public/alerting/preview_alert_condition.tsx

```diff
@@ -88,7 +88,18 @@ const AlertInstancePreview: FC<PreviewResponse['results'][number]> = React.memo(
                       <EuiCode transparentBackground>
                         {i.function}({i.field_name})
                       </EuiCode>{' '}
-                      {i.by_field_value} {i.over_field_value} {i.partition_field_value} [{i.score}]
+                      {i.by_field_value} {i.over_field_value} {i.partition_field_value} [{i.score}];
+                      (
+                      <FormattedMessage
+                        id="xpack.ml.previewAlert.typicalLabel"
+                        defaultMessage="Typical:"
+                      />{' '}
+                      {i.typical ?? '-'},{' '}
```

Comment:
Not needed for this PR, but a nice-to-have would be to use the Kibana field formatter for displaying these values in the preview and alert, for example formatting as bytes here:

![image](https://user-images.githubusercontent.com/7405507/143072482-58854b57-022a-4c45-aa02-14e64d979586.png)

![image](https://user-images.githubusercontent.com/7405507/143072678-926660ac-7aef-4582-a775-79bc179429f9.png)

Reply:
Done in https://github.com/elastic/kibana/pull/118006/commits/f62391de411aea39567f6c1190ccf26ece299364